### PR TITLE
twitter waitlist confirmation email

### PR DESCRIPTION
**Pull request by ahsu1230**
@andrewconner @stkem @joshm1

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/views/email/black/twitterWaitlist.scala.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+@(toUserId: Id[User])
+@import com.keepit.common.mail.template.helpers._
+<table width="100%" border="0" cellspacing="0" cellpadding="0" align="center" style="border-collapse: collapse; mso-table-lspace: 0px; mso-table-rspace: 0px;">
```
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/app/views/email/black/twitterWaitlist.scala.html:3]: email is still missing a lot of pieces for now, but will add them in separate PRs to keep this current one smaller

File: kifi-backend/modules/shoebox/app/views/email/black/twitterWaitlist.scala.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <tr><td height="18" style="line-height: 1px; font-size: 1px; text-align: center; border-collapse: collapse; mso-line-height-rule: exactly;">&nbsp;</td></tr>
+    <tr>
+        <td class="content" style="font-family: 'Helvetica Neue', Helvetica, Arial , sans-serif; font-size: 19px; color: #505E65; text-align: center; border-collapse: collapse; mso-line-height-rule: exactly;" align="center">
+            We will let you know as soon as your<br>
+            Kifi Twitter library is ready
+        </td>
+    </tr>
+    <tr>
+        <td height="18" style="line-height: 1px; font-size: 1px; border-collapse: collapse; text-align: center; mso-line-height-rule: exactly;">
+            In the meantime, be sure to tweet about Kifi <br>
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/views/email/black/twitterWaitlist.scala.html:20]: Let's double check this copy. There's no CTA here — no hashtag, no link. A bit weird.

File: kifi-backend/modules/shoebox/app/views/email/black/twitterWaitlistText.scala.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+@(toUserId: Id[User])
+@import com.keepit.common.mail.template.helpers._
+
+Hey @firstName(toUserId), You are on the list
+
+We will let you know as soon as your Kifi Twitter library is ready
+
+In the meantime, be sure to tweet about Kifi Twitter Deep Search in order to move up on the list.
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/views/email/black/twitterWaitlistText.scala.html]: Same here.

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/TwitterWaitlistEmailSender.scala [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    protected val airbrake: AirbrakeNotifier) extends Logging {
+
+  def sendToUser(email: EmailAddress, userId: Id[User]): Future[ElectronicMail] = {
+    val emailToSend = EmailToSend(
+      title = "kifi — Congrats! You're on the list",
+      fromName = Some(Right("Kifi")),
+      from = SystemEmailAddress.NOTIFICATIONS,
+      to = Right(email),
+      subject = "You're on the list",
+      category = NotificationCategory.User.TWITTER_WAITLIST,
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/TwitterWaitlistEmailSender.scala]: How about using `WAITLIST` instead? This is basically a one-time use feature as we roll this out.

**Pull request merged**